pkgs/development/tools/py-spy/default.nix

```diff
@@ -0,0 +1,30 @@
+{ stdenv, pkgsBuildBuild, rustPlatform, fetchFromGitHub, pkgconfig, libunwind }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "py-spy";
+  version = "0.3.3";
+
+  src = fetchFromGitHub {
+    owner = "benfred";
+    repo = "py-spy";
+    rev = "v${version}";
+    sha256 = "1w9nwsmazafr78ghif6njdcqjisr0f1jb6cm1w4ngvigx5qlrbkv";
+  };
+
+  NIX_CFLAGS_COMPILE = "-L${libunwind}/lib";
+
+  # error: linker `arm-linux-gnueabihf-gcc` not found
+  preConfigure = stdenv.lib.optionalString (stdenv.buildPlatform != stdenv.hostPlatform) ''
+    export RUSTFLAGS="-Clinker=$CC"
+  '';
+
+  doCheck = false;
```

Comment:
Can you comment why you disabled the tests? Take they to long?

Reply:
I needed a cross build so didn't want to deal with the tests pulling in extra stuff, but turns out they just need a  python binary.